content/features/events/index.md

```diff
@@ -0,0 +1,40 @@
+---
+uid: dnn-features
```

Comment:
@mikesmeltzer I suggest `uid: event-system`

Reply:
@mikesmeltzer hey - just checking back in on this quick change request.  Let me know if you need to discuss it in more detail before proceeding.  Thanks!

Reply:
@david-poindexter This makes sense to me! I'll look to fix this up later today. Thanks :)

Reply:
@mikesmeltzer any status updates on this?  Thanks.

Reply:
@david-poindexter I think the changes are all made now. Back to you sir :)

Reply:
@mikesmeltzer thank you sir!